process/ABProcessParticipantCore.js

```diff
@@ -83,6 +83,12 @@ module.exports = class ABProcessParticipantCore extends ABMLClass {
          this.useField = parseInt(attributes.useField);
       }
 
+      this.userField = [];
+      if (typeof attributes.userField != "undefined") {
```

Comment:
Is it better if we uses
`attributes.userField != null`

because `typeof null` -> `'object'`